src/loki/chart/templates/loki-schema.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: loki-schema 
+  namespace: {{ .Release.Namespace }}
+data:
+  # v13schemadate defaults to two days in the future
+  {{- $configmap := (lookup "v1" "ConfigMap" "loki" "loki-schema") }}
+  {{- if .Values.v13schemadate }}
+  v13schemadate: {{ .Values.v13schemadate }}
+  {{- else if $configmap }}
+  v13schemadate: {{ $configmap.data.v13schemadate }}
+  {{- else }}
+  v13schemadate: {{ (now | dateModify "+48h" | date "2006-01-02") | quote }}
+  {{- end }}
```

Comment:
(realize that I suggested some of this, but a few thoughts)

- Not really sure we need `v13schemadate` value - if someone wants to override they could directly modify the loki schema values, although they'd have to override the full block
- If this is a fresh install we can actually put the date in the past - we don't have to, but it's one less thing for new installs to have to include. Probably could do with [`.Release.IsInstall`](https://helm.sh/docs/chart_template_guide/builtin_objects/) + dateModify minus 48h?
- Controlling this schema config with helm templating makes me a bit scared, but it may be the right place? Can see both sides to this...